module/src/main/resources/zipkin-server-aws.yml

```diff
@@ -0,0 +1,56 @@
+# When enabled, this allows shorter env properties (ex -Dspring.profiles.active=aws)
+spring:
+  main:
+    # ZipkinElasticsearchAwsStorageModule overrides the bean named 'storage'
+    allow-bean-definition-overriding: true
+zipkin:
+  internal:
+    module:
+      elasticsearch-aws: zipkin.module.storage.aws.elasticsearchZipkinElasticsearchAwsStorageModule
```

Comment:
these are still broken out, just for convenience. the profile itself is no longer sensitive to contents.